client/package.json

```diff
@@ -1,9 +1,12 @@
 {
     "name": "superdesk",
     "license": "GPL-3.0",
+    "scripts": {
+        "postinstall": "cd extensions/belgaCoverage && npm install && npm run compile"
```

Comment:
I think it would be better to have this in belgaCoverage package.json file, here add belgaCoverage to dependencies:
```js
"belga-coverage": "file:./extensions/belgaCoverage",
```
and make the import in index like `import belgaCoverageExtension from 'belga-coverage'`